src/libraries/System.Diagnostics.EventLog/src/GenerateMessages.targets

```diff
@@ -0,0 +1,68 @@
+<Project>
```

Comment:
Maybe it’s simpler if I just factor this into a separate project. I think that’ll require much less customization.

Reply:
I did this and it's much simpler :)